lib/twitter.js

```diff
@@ -0,0 +1,87 @@
+const querystring = require('querystring');
+const Twit = require('twit');
+const envVars = require('../env');
+
+const getClient = () => new Twit({
+  consumer_key: envVars.TWITTER_CONSUMER_KEY,
+  consumer_secret: envVars.TWITTER_CONSUMER_SECRET,
+  access_token: envVars.TWITTER_ACCESS_TOKEN,
+  access_token_secret: envVars.TWITTER_ACCESS_TOKEN_SECRET,
+});
+
+const sleep = (seconds = 5) => new Promise((resolve) => setTimeout(
+  resolve,
+  seconds * 1000,
+));
+
+const search = async (
+  client,
+  options,
+  iteration,
+  maxIterations,
+) => {
+  const thisIteration = iteration + 1;
+  const response = await client.get('search/tweets', options);
+  if (response.data && response.data.statuses) {
+    const tweets = response.data.statuses;
+    if (
+      thisIteration < maxIterations
+      && tweets.length === Number(options.count)
+      && response.data.search_metadata
+      && response.data.search_metadata.next_results
+    ) {
+      await sleep();
+      const { next_results: url } = response.data.search_metadata;
+      const nextTweets = await search(
+        client,
+        querystring.parse(url.replace(/^[\?&]/, '')),
+        thisIteration,
+        maxIterations,
+      );
+
+      tweets.push(...nextTweets);
+    }
+
+    return tweets;
+  }
+
+  return [];
+};
+
+const searchTweets = async ({
+  client = null,
+  query = '',
+  count = 100,
+  sinceId = null,
```

Comment:
Have you tried using `undefined` here? That might allow you to get rid of lines 72 through 74 and have a single `if` for `sinceId`, since the request would probably filter it out.